src/main/resources/migrations/0001_create_table_webhooks.sql

```diff
@@ -0,0 +1,18 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:create-table-webhooks
+
+CREATE TYPE live_or_test AS ENUM('LIVE', 'TEST');
+
+CREATE table webhooks (
+    id SERIAL PRIMARY KEY,
+    created_at TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+    external_id VARCHAR(30) NOT NULL,
+    service_id VARCHAR(30) NOT NULL,
+    type live_or_test NOT NULL,
```

Comment:
any reason we went enum over boolean here (given we went the other way in connector and ledger)? Are we going to hate ourselves when we find ourselves writing enum deserialisers all over the shop in a weeks time?